crates/starknet-devnet-types/src/rpc/estimate_message_fee.rs

```diff
@@ -14,6 +15,17 @@ use crate::felt::Felt;
 use crate::rpc::eth_address::EthAddressWrapper;
 use crate::{impl_wrapper_deserialize, impl_wrapper_serialize};
 
+// TODO: maybe move this to new file?
```

Comment:
Let's resolve this TODO before merging.

Reply:
Moved to new file in bf453f7bbdcc93dfed942527865acaa76cb08d1a